code/game/machinery/teleporter.dm

```diff
@@ -196,7 +196,7 @@
 	name = "teleporter horizon generator"
 	desc = "This generates the portal through which you step through to teleport elsewhere."
 	icon_state = "tele0"
-	var/accurate = 0
+	var/accurate = 0 //Doesn't compile without it due to legacy stuff. Unused otherwise.
```

Comment:
Man, just fix the problem rather than leaving it here. You don't even have to use the mapper. You can just remove the references with a text editor.